features/step_definitions/scheduler.rb

```diff
@@ -0,0 +1,19 @@
+Given /^the #{QUOTED} scheduler CRD is restored after scenario$/ do |name|
```

Comment:
CR looks more accurate :), CR is Resource, CRD is Definition :)

Reply:
Have updated.